component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/SpaceMemberDAOImpl.java

```diff
@@ -236,4 +236,17 @@ public int countPendingSpaceRequestsToManage(String username) {
     return query.getSingleResult().intValue();
   }
 
+  public int countExternalMembers(Long spaceId) {
+    Query q = getEntityManager().createNativeQuery("SELECT COUNT(USER_ID) FROM SOC_SPACES_MEMBERS ssm " +
```

Comment:
What about the fact user can be member AND manager in a space ? 
Does the COUNT(USER_ID) answers 2 ? 
Should we count only distinct user_id ?

Reply:
added distinct